text-serializer-gson/src/main/java/net/kyori/adventure/text/serializer/gson/GsonComponentSerializer.java

```diff
@@ -101,8 +101,13 @@ static Builder builder() {
     @NonNull Builder legacyHoverEventSerializer(final @NonNull LegacyHoverEventSerializer serializer);
 
     /**
-     * Output a legacy hover event {@code value} in addition to the modern {@code contents}
-     *
+     * Clears the serializer used to interpret and emit legacy hover event {@code value} payloads.
+     * @return this builder
+     */
+    @NonNull Builder clearLegacyHoverEventSerializer();
```

Comment:
I think it might be better to just make `legacyHoverEventSerializer` accept a `@Nullable` parameter instead?

Reply:
That's reasonable. I can make this change.